iceoryx_hoofs/test/moduletests/test_cxx_unique_ptr.cpp

```diff
@@ -161,27 +160,10 @@ TEST_F(UniquePtrTest, ReleaseAnObjectResultsInDeleterNotBeingCalled)
     ::testing::Test::RecordProperty("TEST_ID", "8a1413a5-15cd-42ff-a05e-9dff158aa047");
     auto sut = iox::cxx::unique_ptr<Position>(&object, deleter);
 
-    EXPECT_EQ(iox::cxx::unique_ptr<Position>::release(std::move(sut)), &object);
-    EXPECT_FALSE(m_deleterCalled);
-}
-
-TEST_F(UniquePtrTest, ResetToAnExistingObjectPtrResultsInDeleterCalledTwice)
```

Comment:
Can we reuse this test for the move assignment operator instead of `reset`?

Reply:
@FerdinandSpitzschnueffler I think the test case you were thinking about is already implemented in `75a853ef-fd0e-41bd-9ce7-af63e0f67fa9`, correct?